stepconf/stepconf.go

```diff
@@ -90,7 +89,7 @@ func setField(field reflect.Value, value, constraint string) error {
 	case reflect.String:
 		field.SetString(value)
 	case reflect.Bool:
-		b, err := parseutil.ParseBool(value)
+		b, err := parseBool(value)
```

Comment:
I decided to inline this. It's only usage other than this is in a deprecated repo: https://cs.github.com/?scope=org:bitrise-io&scopeName=bitrise-io&q=parseutil.ParseBool